ui/design-system-core/src/components/DuplexButton/index.tsx

```diff
@@ -44,6 +44,7 @@ const DuplexButton = (props: DuplexButtonProps) => {
   const [iconOnly, setIconOnly] = useState(window.matchMedia('(max-width: 992px)').matches);
   const activeHoverIconElem = activeHoverIcon ?? icon;
   const activeIconElem = activeIcon ?? icon;
+  const fixWidth = 'w-[7rem]';
```

Comment:
How about setting a minimum width here rather than a fixed width or how about using a fixedWidth prop to make a conscious decision to use fixed width ? Otherwise if at some point we have a `DuplexButton` that needs to extend beyond `7rem` then this wont work unless we want to wrap text.